tests/test_cli.py

```diff
@@ -15,25 +15,23 @@
 import rsa
 import rsa.cli
 import rsa.util
+from rsa._compat import PY3
 
-if sys.version_info[0] < 3:
-    def make_buffer():
-        return BytesIO()
 
-
-    def get_bytes_out(out):
-        # Python 2.x writes 'str' to stdout:
-        return out.getvalue()
-else:
-    def make_buffer():
+def make_buffer():
```

Comment:
It makes sense, and I agree that readability is very important. However, I'm also curious as to the runtime performance impact. Do you have time to do some timing tests with some large key generation and file encryption?